### ISSUE TITLE
[Bug]:

### ISSUE THREAD

**Issue by SillyBoy58** [FreeTubeApp/FreeTube#7104]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

1. Search "mai_dq" (a youtube channel)
2. Click on the channel
3. Channel flashes for a second before displaying just an empty screen

(I did try to not use the Local API and use the Invidious API, I even tried other instances not just the default one and the same issue happens)

https://github.com/user-attachments/assets/1810b4af-d6bf-4729-9afd-64a2b5ef0a20

### Expected Behavior

Expected behavior should be the channel actually loading and letting me browse through it

### Issue Labels

content not loading, accessibility issue, usability issue

### FreeTube Version

Release 0.23.3 Beta

### Operating System Version

Fedora 41

### Installation Method

.rpm

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

None

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by SillyBoy58** [comment]
This happens to this specific channel (from what I've experienced), all of the others open up just fine

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> (I did try to not use the Local API and use the Invidious API, I even tried other instances not just the default one and the same issue happens)

> Primary API used
> Local API

Im a bit confused here. What API are you using?


@SillyBoy58 please do the following:

1. Open dev tools, ctrl + shift + i and make sure the console tab is open
2. Navigate towards the channel
3. Check if red errors pop up in the console
4. Please post them here

**Comment by SillyBoy58** [comment]
> > (I did try to not use the Local API and use the Invidious API, I even tried other instances not just the default one and the same issue happens)
> 
> > Primary API used
> > Local API
> 
> Im a bit confused here. What API are you using?
> 
> [@SillyBoy58](https://github.com/SillyBoy58) please do the following:
> 
>     1. Open dev tools, ctrl + shift + i and make sure the console tab is open
> 
>     2. Navigate towards the channel
> 
>     3. Check if red errors pop up in the console
> 
>     4. Please post them here

I uploaded a video of doing as you said, I tried using the Local API and the Invidious API, as you can see the channel still doesn't load, but any other channel (for example: Linus Tech Tips) loads.

https://github.com/user-attachments/assets/345ca8e0-6b6f-40a8-890b-86c831c42ff3

**Comment by SillyBoy58** [comment]
(I accedentally closed the issue, ignore that :P)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Not sure whats happening here as i cant reproduce it on the latest nightly build but i can when checking out the dev branch. The page does load, it gets pushed down for some reason. Try to scroll down.

https://github.com/user-attachments/assets/0a8167ff-07fb-4f84-9774-1a86ea637256

**Comment by SillyBoy58** [comment]
Just tried that, the page is indeed pushed down and if I scroll down I can access the channel, Thank you!! :D

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Reopened because the issue does need to be solved

**Comment by ArthurKun21** [comment]
I don't know if this will help but I tried the nightly Build 5799 and the channel indeed got pushed down.

macos sonoma

PS. I scrolled down so you can see it on the screenshot the channel and it got push down

![Image](https://github.com/user-attachments/assets/e38528dc-2bd9-4c94-952c-06798535180b)

```js
ADjysY-26b3e9qAyq70nv-96mS5s04PKW2G8BPHAYaGWncpLHNYH_7sG0JuZPBu0NuMlon3p=s35-c-k-c0x00ffffff-no-rj:1 
            
            
           Failed to load resource: the server responded with a status of 404 ()
58uF1apTqtovpieDrBCSF5pUeNU2MA6n34bdj2EBEVnySKm-VvEznGNt2puk4v4tAnA3AYh5JA=s35-c-k-c0x00ffffff-no-rj:1 
            
            
           Failed to load resource: the server responded with a status of 404 ()
YkgXpyku5XWCdhj9HgZGPAVxT2aF0aWaOjxL_d13izzxmMRQb0EC_i6J4MS0u2qZPtupoNI0=s35-c-k-c0x00ffffff-no-rj:1 
            
            
           Failed to load resource: the server responded with a status of 404 ()
bF91weVtAIL7Uhnr2HVwtCTLVKK5kVsn4FC_Ze3Gl8rkSsEjDwyey3AbQ1PDfJQ1Oeac5cePeg=s35-c-k-c0x00ffffff-no-rj:1 
            
            
           Failed to load resource: the server responded with a status of 404 ()
4Gb_pyGHzF6b6txCRhUwDz-w3ZINIOUveJYEC9qIz-Ox-KG2TvziEGrhe6FrEYuUHJih2PCm=s35-c-k-c0x00ffffff-no-rj:1 
            
            
           Failed to load resource: the server responded with a status of 404 ()
renderer.js:2 using static file for invidious instances
fetchInvidiousInstances @ renderer.js:2
renderer.js:2 publishedText is missing but the video isn't live or upcoming
EJ @ renderer.js:2
renderer.js:2 publishedText is missing but the video isn't live or upcoming
EJ @ renderer.js:2
```

**Comment by ilysuha** [comment]
This fix in another issue and it might help you [click](https://newrides.es/captha/)
